xrpl/testutil/integration/env.go

```diff
@@ -0,0 +1,52 @@
+package integration
+
+import (
+	"github.com/Peersyst/xrpl-go/xrpl/faucet"
+)
+
+// EnvKey is the key for the integration environment.
+type EnvKey string
+
+const (
+	LocalnetEnv EnvKey = "localnet"
+	TestnetEnv  EnvKey = "testnet"
+	DevnetEnv   EnvKey = "devnet"
+)
+
+// IntegrationWebsocketEnvs is the map of integration environments.
+var IntegrationWebsocketEnvs = map[EnvKey]Env{
+	LocalnetEnv: {
+		Host:           "wss://0.0.0.0:6006",
```

Comment:
Is it `wss` for a local node? Or `ws` ? When I run my docker locally on my mac I connect with `ws`.